deb/deb.go

```diff
@@ -90,7 +91,7 @@ func (*Deb) Package(info *nfpm.Info, deb io.Writer) (err error) { // nolint: fun
 		return err
 	}
 
-	dataTarGz, md5sums, instSize, err := createDataTarGz(info)
+	dataTarGz, md5sums, instSize, dataTarballName, err := createDataTarball(info)
```

Comment:
`dataTarGz` could be named `dataTarball` now I think

Reply:
Thanks, I found a lot of other places too where I did not update the naming. It should all be consistent now.